packages/performance/index.ts

```diff
@@ -36,6 +36,10 @@ export function registerPerformance(instance: FirebaseNamespace): void {
     if (app.name !== DEFAULT_ENTRY_NAME) {
       throw ERROR_FACTORY.create(ErrorCode.FB_NOT_DEFAULT);
     }
+    if (!window) {
```

Comment:
It will still throw exception in Nodejs. `typeof window !== undefined` should work.

Reply:
Good point! typeof returns a string. Updated the condition.